### PR TITLE
Add Image Classification Blog Post

### PR DESCRIPTION
**Pull request by PineappleIOnic**
## What does this PR do?

This PR adds the image classification blog post

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Review by christyjacob4 — APPROVED** [review]

File: src/routes/blog/post/image-classification/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+All these efforts culminated in 2012 when AlexNet was showcased at the ImageNet Large Scale Visual Recognition (ILSVRC), winning it by a significant margin. AlexNet's success showcased what was possible using Convolutional Neural Networks (CNNs) for image classification and sparked a seismic shift in how we approach image classification going forward.
+
+Following AlexNet, several architectures were released that built upon what AlexNet had achieved. A few notable examples include VGGNet and GoogLeNet in 2014, ResNet in 2015, and EfficientNet in 2019. Each further refined and simplified the CNN Architecture that is widely used today.
+
+Several more trends are emerging in the future. One of these includes utilizing retention mechanisms and transformers, inspired by their massive success in natural language processing, where models treat image patches as tokens, similar to how we treat words in text. Few-shot learning alongside transfer learning has also been explored to reduce the reliance on huge datasets and make training these models more efficient.
+
+## Architectures
```
Comment (christyjacob4) [inline src/routes/blog/post/image-classification/+page.markdoc:26]: Maybe you can use H1s here?  and for #Datasets and examples

File: src/routes/blog/post/image-classification/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+```bash
+appwrite init project // Use if you don't already have an appwrite.json
+appwrite init function
+```
+
+Give it any name and make sure to select the `Python (ML)` runtime; this should create a new function with the boilerplate included. Change directories into the freshly created function and open a code editor within it.
+
+### Step 3: Add our inference code to the function
+
+With the function boilerplate created, we will replace the `main.py` file with a script we'll run on our function to load the model we just trained and make evaluations based on it. Replace all the code in that file with the following code:
```
Comment (loks0n) [inline src/routes/blog/post/image-classification/+page.markdoc:163]:
```suggestion
With the function boilerplate created, replace the `main.py` file with a script to load the model we just trained and run inference:
```

File: src/routes/blog/post/image-classification/+page.markdoc [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+
+    response_message = "This image most likely belongs to {} with a {:.2f} percent confidence.".format(
+        class_names[np.argmax(score_lite)], 100 * np.max(score_lite)
+    )
+
+    context.log(response_message)
+    return context.res.send(response_message, 200)
+```
+
+We also want to add a requirements.txt file with TensorFlow and NumPy, so we have all the dependencies we need:
```
Comment (loks0n) [inline src/routes/blog/post/image-classification/+page.markdoc:234]:
```suggestion
Add a requirements.txt file with TensorFlow and NumPy, to fetch the required dependencies:
```

File: src/routes/blog/post/image-classification/+page.markdoc [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+```text
+tensorflow
+appwrite
+numpy
+Pillow
+```
+
+### Step 4: Deploy your function
+
+Next, move the model.tflite file that was created into the same directory as your main.py file and then change directories to where your appwrite.json is stored. This model.tflite is quite a small file, so we can store it in the function. If you’re using larger models, upload them to Appwrite Storage and then use getFileDownload to download them to your function.
```
Comment (loks0n) [inline src/routes/blog/post/image-classification/+page.markdoc:245]:
```suggestion
Move the `model.tflite` file that was created into the same directory as the `main.py` file and then change directories to that containing the `appwrite.json`. This `model.tflite` is quite a small file, so store it in the function. If using larger models, upload them to Appwrite Storage and then use `getFileDownload` to download them to the function. 
```

File: src/routes/blog/post/image-classification/+page.markdoc [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+appwrite
+numpy
+Pillow
+```
+
+### Step 4: Deploy your function
+
+Next, move the model.tflite file that was created into the same directory as your main.py file and then change directories to where your appwrite.json is stored. This model.tflite is quite a small file, so we can store it in the function. If you’re using larger models, upload them to Appwrite Storage and then use getFileDownload to download them to your function. 
+
+With the file now copied, run appwrite deploy function and select the function you just created to deploy it to Appwrite.
```
Comment (loks0n) [inline src/routes/blog/post/image-classification/+page.markdoc:247]:
```suggestion
With the file now copied, run appwrite deploy function and select the function to deploy it to Appwrite.
```

File: src/routes/blog/post/image-classification/+page.markdoc [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+
+### Step 5: Update your environment variables 
+
+Now that the function has been deployed, navigate to it on the Appwrite console and set the `APPWRITE_API_KEY` and `APPWRITE_BUCKET_ID` environment variables to be ready for execution. Make sure to redeploy the function so these new environment variables are set.
+
+### Step 6: Test your function
+
+Finally, send a `POST` request to your function with a JSON body that contains a file ID from your bucket like this:
```
Comment (loks0n) [inline src/routes/blog/post/image-classification/+page.markdoc:255]:
```suggestion
Finally, send a `POST` request to the function with a JSON body that contains a file ID from the bucket like this: 
```

File: src/routes/blog/post/image-classification/+page.markdoc [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+
+### Capsule Networks (CapNet)
+
+[CapNets](https://arxiv.org/pdf/1710.09829) were invented by Geoffrey Hinton to remedy CNN's issue of recognizing different orientations. Unlike CNNs, CapNets don't use max-pooling techniques, which may cause important details to be lost in an image. Instead, they utilize smaller groups of neurons called capsules, designed to detect specific objects and output a vector instead of a single float value. Because of this output, CapNets can understand relationships between objects better (Eyes appear above the nose, etc.…) to build up a more accurate classifier even when viewing images with different orientations.
+
+# Datasets
+
+Finding data to train these architectures can sometimes be the most challenging part of developing state-of-the-art image classification models. It was tough in the early days, but these days, we have standardized high-quality labeled data that is offered entirely free for any researchers who require it. Next, I'll cover some of the most popular datasets.
```
Comment (loks0n) [inline src/routes/blog/post/image-classification/+page.markdoc:47]:
```suggestion
Finding data to train these architectures used to b the most challenging part of developing state-of-the-art image classification models. Now, however, there is standardized high-quality labeled data that is offered entirely free for any researchers who require it. Next, we'll cover some of the most popular datasets.
```

**Review by loks0n — CHANGES_REQUESTED** [review]

**Review by loks0n — APPROVED** [review]

File: src/routes/blog/post/image-classification/+page.markdoc [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+title: A technical deep dive into image classification
+description: Learn all about image classification and how to integrate it into your Appwrite project using TensorFlow
+date: 2024-08-06
+cover: /images/blog/image-classification.png
+timeToRead: 9
+author: bradley-schofield
+category: AI
+---
+
+Image Classification is an exciting field of Computer Vision that tries to understand and label images in their entirety. It has many applications: you can integrate it into your app to automatically categorize photos, filter inappropriate content from social media feeds, or even organize your online store's product catalog.
```
Comment (loks0n) [inline src/routes/blog/post/image-classification/+page.markdoc:12]:
```suggestion
Image classification is an exciting field of computer vision that tries to understand and label images in their entirety. It has many applications: you can integrate it into your app to automatically categorize photos, filter inappropriate content from social media feeds, or even organize your online store's product catalog.
```

File: src/routes/blog/post/image-classification/+page.markdoc [hunk 9]
```diff
@@ -0,0 +1,270 @@
...
+
+In this article, we’ll go over how the image classification field developed and its most popular architectures and datasets. Then, we’ll run you through implementing it into your Appwrite project using TensorFlow.
+
+# A Brief History of Image Classification
```
Comment (loks0n) [inline src/routes/blog/post/image-classification/+page.markdoc:16]:
```suggestion
# A brief history of image classification
```

**Review by loks0n — APPROVED** [review]
spotted a few titles that should be sentence case.

LGTM otherwise

**Pull request merged**